cmd/sealer/cmd/run.go

```diff
@@ -76,12 +89,66 @@ create a cluster with custom environment variables:
 		}
 
 		cluster := cf.GetCluster()
+
 		infraDriver, err := infradriver.NewInfraDriver(&cluster)
 		if err != nil {
 			return err
 		}
 
-		//TODO mount image and copy to cluster
+		hostsPlatformMap, err := infraDriver.GetHostsPlatform(infraDriver.GetHostIPList())
+		if err != nil {
+			return err
+		}
+
+		distributor, err := imagedistributor.NewScpDistributor(infraDriver)
+		if err != nil {
+			return err
+		}
+
+		imageEngine, err := imageengine.NewImageEngine(imagecommon.EngineGlobalConfigurations{})
+		if err != nil {
+			return err
+		}
+
+		var (
+			clusterImageName  = cluster.Spec.Image
+			rootfs            = infraDriver.GetClusterRootfs()
+			rootfsRegistryDir = filepath.Join(rootfs, "registry")
+		)
+
+		for platform, hosts := range hostsPlatformMap {
```

Comment:
hostsPlatformMap should be handle by imageEngine and imageDistributor, not cmd run.